sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/keyvault/secrets/properties/AzureKeyVaultPropertySourceProperties.java

```diff
@@ -31,11 +31,11 @@ public class AzureKeyVaultPropertySourceProperties extends AbstractAzureHttpConf
      */
     private String name;
     /**
-     * Defines the constant for the property that enables/disables case-sensitive keys.
+     * Whether to enable case-sensitive for secret keys. The default value is `false`.
      */
     private boolean caseSensitive = false;
     /**
-     * The secret keys supported for this property source.
+     * The configured secret keys will be loaded from Azure Key Vaults secret, if configured nothing, then load all the secrets.
```

Comment:
Do we need to mention the format here? Will it use the exact secret names stored in the KV or using Spring format?

Reply:
I think you get the point, we definitely should provide such info, or customer may wonder what key they should provide to this property